sonar-kotlin-plugin/build.gradle.kts

```
@@ -96,8 +96,8 @@ val javadocJar = tasks.javadocJar
 tasks.shadowJar {
     minimize {}
     exclude("META-INF/native/**/*jansi*")
-    exclude("org/jetbrains/kotlin/org/jline/**")
-    exclude("org/jetbrains/kotlin/net/jpountz/**")
+    exclude("org/jline/**")
+    exclude("net/jpountz/**")
     doLast {
         enforceJarSizeAndCheckContent(shadowJar.get().archiveFile.get().asFile, 38_100_000L, 38_600_000L)
```

Indeed. Thanks for noticing this. Looks like it also affected the size, so the boundaries should be lowered
```suggestion
        enforceJarSizeAndCheckContent(shadowJar.get().archiveFile.get().asFile, 37_500_000L, 38_000_000L)
```